### PR TITLE
Us151 deed not found

### PR DESCRIPTION
**Pull request by Phemus**
Implements new deed not found error message on enter token page and redirects to this page if details entered by user do not match

### CODE REVIEW

File: application/deed/searchdeed/views.py [hunk 1]
```diff
@@ -10,7 +10,9 @@
 
 @searchdeed.route('/')
 def search_deed_main():
-    return render_template('searchdeed.html')
+    display_error = None
+
+    return render_template('searchdeed.html', error=display_error)
```
Comment (Kirosoft) [inline application/deed/searchdeed/views.py]: error=None - remove the variable

**Comment by Kirosoft** [comment]
:+1:

**Comment by endamccormack** [comment]
:+1:

**Comment by tsangsimon** [comment]
:+1:

**Pull request merged**